crates/core/tedge_mapper/src/az/mapper.rs

```diff
@@ -39,7 +42,11 @@ impl TEdgeComponent for AzureMapper {
         Ok(())
     }
 
-    async fn start(&self, tedge_config: TEdgeConfig) -> Result<(), anyhow::Error> {
+    async fn start(
+        &self,
+        tedge_config: TEdgeConfig,
+        _config_dir: &PathBuf,
```

Comment:
Why do we need `_config_dir` here?

Reply:
Because, `start` is defined in a `trait`, and it's being implemented by all mappers.  We use it or not, has to pass that parameter.